hooks/collector.py

```diff
@@ -364,17 +399,35 @@ def _get_item_info(self, path):
             icon_path=icon_path,
         )
 
-    def _get_icon_path(self, icon_name):
+    def _get_icon_path(self, icon_name, icons_folder=None):
```

Comment:
Should `icons_folder` be a list of search paths instead? That feels like good flexibility

Reply:
done!